docs/client/api-box.html

```diff
@@ -1,16 +1,28 @@
 <template name="apiBoxTitle">
   <h3 id="{{id}}" class="api-title">
     <a class="name selflink" href="{{link}}">{{{name}}}</a>
-    {{#if locus}}
-      <span class="locus">{{locus}}</span>
-    {{/if}}
+    <div class="locus">
+        {{#if locus}}
+        <span>
+          {{locus}}
+        </span>
+        {{/if}}
+
+        {{#if filepath}}
+        <div class="src-code">
+            <a href="https://github.com/meteor/meteor/blob/master/packages/{{filepath}}#L{{lineno}}" target="_blank">
```

Comment:
Just one thing : because jsdoc returns the function's position in the currently working branch, and the source code url links to the `master` branch, the url might points to the wrong position. But I think it's safe to assume no one will deploy locally the doc app just to check out the links, so this might not be a big deal.